README.md

```diff
@@ -150,6 +150,48 @@ should be configured to neither warn about nor overwrite _Black_'s changes.
 Actual details on _Black_ compatible configurations for various tools can be found in
 [compatible_configs](https://github.com/psf/black/blob/master/docs/compatible_configs.md).
 
+### Migrating your code style without ruining git blame
+
+A long-standing counterargument against moving to automated code formatters like _Black_
+is that the migration will clutter up the output of `git blame`. This used to be a valid
+argument, but not anymore. Since `git` version 2.23, git natively supports
+[ignoring revisions in blame](https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revltrevgt)
+with the `--ignore-rev` option. You can also pass a file listing the revisions to ignore
+using the `--ignore-revs-file` option. The changes made by the revision will be ignored
+when assigning blame. Lines modified by an ignored revision will be blamed on the
+previous revision that modified those lines.
+
+So when migrating your project's code style to _Black_, reformat everything and commit
+the changes (preferably in one massive commit). Then put the full 40 characters commit
+identifier(s) into a file.
+
+```
+# Migrate code style to Black
+5b4ab991dede475d393e9d69ec388fd6bd949699
+```
+
+Afterwards, you can pass that file to `git blame` and see clean amd meaningful blame
+information.
+
+```
+$ git blame important.py --ignore-revs-file .git-blame-ignore-revs
+7a1ae265 (John Smith 2019-04-15 15:55:13 -0400 1) def very_important_function(text, file):
+abdfd8b0 (Alice Doe  2019-09-23 11:39:32 -0400 2)     text = text.lstrip()
+7a1ae265 (John Smith 2019-04-15 15:55:13 -0400 3)     with open(file, "r+") as f:
+7a1ae265 (John Smith 2019-04-15 15:55:13 -0400 4)         f.write(formatted)
+```
+
+You can even configure `git` to automatically ignore revisions listed in a file on every
+call to `git blame`.
+
+```shell
+$ git config blame.ignoreRevsFile .git-blame-ignore-revs
+```
+
+**The one caveat is that GitHub and GitLab do not yet support ignoring revisions using
+in their native UI of blame.** So blame information will be cluttered with a reformating
+commit on those platforms.
```

Comment:
Is there a public issue or request anywhere we can link to for people to +1? 

If not, maybe we should kick it off someway or at least via my own paid personal account.

Reply:
Found an issue for GitLab.. still looking for GitHub
https://gitlab.com/gitlab-org/gitlab/-/issues/31423

Reply:
Probably doesn't count, but here's something.
https://github.community/t5/How-to-use-Git-and-GitHub/Support-ignore-revs-file-in-Github-s-blame-view/m-p/57020

Reply:
It appears like the ways to submit feature requests regarding github.com is either via https://support.github.com/contact/feedback which isn't public, or on the GitHub Community Forum.

Quoted from this answer for a question about submitting feature questions regarding GitHub Community Forum:
>That said, I do want to share that we are working on a new tool and process for submitting these sorts of things for the Community Forum (and for github.com). This is still in the investigative stage, and I don't have a timeline to share, but we are planning on improving this process.
>
>We also recognize the desire for feature requests to be visible to other users, commented on, and even voted on. These are all things that we are taking into consideration when looking at implementing a new process.

Reply:
Also, is my documentation a bit confusing? You wrote '.git-blame-ignore-revs support', that's just the file name that is used by convention for the file containing the revisions to ignore. It's nothing special.

Anyway, @cooperlees, do you think those two links work? If they do, I'll add them in the docs.

Reply:
Yeah sounds good - Lets just add the link for GitLab + say if you'd like this feature please let GitHub know!

Re being confusing ... no ... I just copy pasted - I should have used `--ignore-revs-file` to refer to the feature.

Reply:
https://github.com/isaacs/github is an unofficial place to post GitHub requests to +1. Wanna make one there?

Reply:
While I like the idea of posting an issue there, for personal reasons, I don't want to post the issue myself... It's not something I'd want to lead, by being the issue author. I hope I am not being ridiculous here in my position.